hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMOpenKeysDeleteRequest.java

```diff
@@ -60,8 +53,21 @@ public class TestOMOpenKeysDeleteRequest extends TestOMKeyRequest {
    */
   @Test
   public void testDeleteOpenKeysNotInTable() throws Exception {
-    OpenKeyBucket openKeys = makeOpenKeys(volumeName, bucketName, 5);
-    deleteOpenKeysFromCache(openKeys);
+    testDeleteOpenKeysNotInTable(BucketLayout.DEFAULT);
```

Comment:
Can we use junit4 test parameterization for the test classes changed in this PR instead of adding a new test method for each bucket layout?